flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -163,9 +163,9 @@
 	 * Non-volatile, because only accessed in synchronized scope */
 	private boolean periodicScheduling;
 
-	/** Flag whether a trigger request could not be handled immediately. Non-volatile, because only
-	 * accessed in synchronized scope */
-	private boolean triggerRequestQueued;
+	/** Flag whether periodic triggering is suspended (too many concurrent pending checkpoint).
```

Comment:
nit: I think the grammar of the commit message is a bit confusing (what was in the past and will be in the future).

If I understand it correctly it should be:
```
This is not just a pure renaming. The semantics has been changed a bit.
If there are too many in-flight checkpoints, the current checkpoint
will fail, and the periodic triggering will be suspended. This stays the same 
as it used to be before.

From now on, there will be no queued semantics. If an in-flight checkpoint
ends its lifecycle, it will try to resume periodic triggering, but no
queued checkpoint will be resumed anymore, as it was in the past.
```
?